builtin/logical/mysql/path_roles.go

```diff
@@ -34,6 +34,11 @@ func pathRoles(b *backend) *framework.Path {
 				Type:        framework.TypeString,
 				Description: "SQL string to create a user. See help for more info.",
 			},
+
+			"username_length": &framework.FieldSchema{
```

Comment:
@jippi You are right about the option becoming redundant. But as long as we don't have something which is applicable to *all*, it can't be elevated to be a mount's option.